scripts/installer/windows/scripts/pfs.bat

```diff
@@ -2,4 +2,12 @@
 setlocal
 
 set MAIN_EXE=%~dp0.\pfcli.exe
-"%MAIN_EXE%" pfs %*
\ No newline at end of file
+REM Check if the first argument is 'start'
+if "%~1"=="start" (
+    cscript %~dp0.\start_pfs.vbs """%MAIN_EXE%"" pfs %*"
+@REM since we won't wait for vbs to finish, we need to wait for the output file to be flushed to disk
+    timeout /t 5 >nul
+    type "%~dp0output.txt"
```

Comment:
what's the expaned path for this?
How do we clean the output.txt file?

Reply:
will redirect the output to sys.executable path. The content will be updated each time we run pfs start

Reply:
![image](https://github.com/microsoft/promptflow/assets/26239730/ceb2df67-a052-4545-9097-6332846da7dc)